src/api/app/javascript/src/turbo_error.js

```diff
@@ -0,0 +1,22 @@
+function renderTurboError(text) {
+  const flash = document.getElementById('flash');
+  const container = flash.querySelector('.col-12');
+  const alert = document.createElement('div');
+  alert.classList.add('alert', 'alert-danger');
+  alert.innerHTML = text;
+  container.appendChild(alert);
+}
```

Comment:
Wouldn't it be easier to have a flash container rendered server-side, even if it's empty / hidden, and then just fill it in below?

Reply:
> That would be always visible and js can create more than one error per page load

Fair enough